library/ssl_tls13_server.c

```diff
@@ -1029,7 +1029,7 @@ static int ssl_tls13_check_ephemeral_key_exchange(mbedtls_ssl_context *ssl)
 }
 
 MBEDTLS_CHECK_RETURN_CRITICAL
-static int ssl_tls13_check_psk_key_exchange(mbedtls_ssl_context *ssl)
+static int ssl_tls13_key_exchange_psk_available(mbedtls_ssl_context *ssl)
 {
 #if defined(MBEDTLS_SSL_TLS1_3_KEY_EXCHANGE_MODE_PSK_ENABLED)
     return ssl_tls13_ticket_permission_check(
```

Comment:
Note: `ssl_tls13_check_psk_key_exchange()` was a `check` function returning non-zero for success, and has been renamed.

However, its implementation also calls a `check` function, and that hasn't been renamed. So we still have `ssl_tls13_ticket_permission_check()` returning non-zero for success